Kite SDK/PSPrintSDK/ViewController.m

```diff
@@ -35,6 +38,55 @@ -(void)viewDidAppear:(BOOL)animated{
 - (void)viewDidLoad {
     [super viewDidLoad];
     
+#ifndef DEBUG
+    self.navigationController.navigationBar.hidden = YES;
```

Comment:
Cool, I'd also move to using the generic iOS Developer / iOS Distribution identities for Code Signing Identity so anyone coming along can build either straight away with no modifications. I think currently release is a Kite Tech LTD specific distribution profile.